app/controllers/physical_server_controller.rb

```diff
@@ -33,7 +33,7 @@ def show_list
   end
 
   def textual_group_list
-    []
+    [%i(properties relationships assets)]
```

Comment:
Cannot test `assets` in the UI since it has too many dependencies that are not merged in yet.
This depends on https://github.com/ManageIQ/manageiq-providers-lenovo/pull/43 which depends on https://github.com/ManageIQ/manageiq/pull/14827.

I recommend taking `assets` out for now until the above PRs are merged... and make this PR about only `properties` and `relationships`